spec/schemas/_common.yaml

```
@@ -2152,31 +2152,31 @@ components:
           type: integer
           format: int32
         total:
-          description: The number of documents that were successfully processed.
+          description: The number of documents that were successfully processed. 
           type: integer
           format: int64
         updated:
-          description: The number of documents that were successfully updated, for example, a document with same ID already existed prior to reindex updating it.
+          description: The number of documents that were successfully updated, for example, a document with same ID already existed prior to the reindex updating it. 
```

@natebower: Lets simplify without the example. How does this sound?

```suggestion
          description: The number of documents that were successfully updated after reindexing.
```

---

That works for me 👍